src/common/options/global.yaml.in

```diff
@@ -4832,6 +4832,14 @@ options:
   desc: Number of additional threads to perform quick-fix (shallow fsck) command
   default: 2
   with_legacy: true
+- name: bluestore_fsck_shared_blob_tracker_size
+  type: size
+  level: dev
+  desc: Size in bytes for a hash table to track shared blobs ref counts. Higher the size, more precise is the tracker -> less overhead during the repair.
+  default: 128_M
+  flags:
+  - runtime
+  with_legacy: true
```

Comment:
It seems to me that we should write this as a transient prioritycache target that competes for the same memory as other targets so it can grow or shrink in the same way as other consumers of memory.  Thoughts?

Reply:
@markhpc - IMO that's an overkill. This is a memory limit for static(!) hash table used in fsck only. And by the nature of this hash table usage I'd prefer it to be predictably sized to avoid side effects and/or false detections.

Reply:
> I think this should be a % of osd_memory_target.

done